guides/04_chatbots/01_creating-a-chatbot-fast.md

```diff
@@ -124,6 +124,28 @@ gr.ChatInterface(
 ).launch()
 ```
 
+## Add Multimodal Capability to your chatbot
+
+You may want to add multimodal capability to your chatbot. For example, you may wantusers to be able to easily upload images or files to your chatbot and ask questions about it. This is supported by the `gr.MultimodalTextbox()` component, which supports both text and file inputs. Here's an example of how to use it:
+
+```python
+import gradio as gr
+
+chat_input = gr.MultimodalTextbox(file_types=["image"], placeholder="Enter message or upload file...")
+```
+
+`gr.ChatInterface` also supports multimodality, simply pass in the `multimodal` parameter as `True`:
```

Comment:
I would remove this part, since this guide is about `ChatInterface`, not the individual components.

```suggestion
```python
import gradio as gr

chat_input = gr.MultimodalTextbox(file_types=["image"], placeholder="Enter message or upload file...")
```

`gr.ChatInterface` also supports multimodality, simply pass in the `multimodal` parameter as `True`:
```